File: core/src/main/java/com/sequenceiq/cloudbreak/controller/json/TemplateResponse.java
package com.sequenceiq.cloudbreak.controller.json;

import com.sequenceiq.cloudbreak.controller.doc.ModelDescriptions;
import com.wordnik.swagger.annotations.ApiModel;
import com.wordnik.swagger.annotations.ApiModelProperty;

@ApiModel
public class TemplateResponse extends TemplateBase {
    @ApiModelProperty(ModelDescriptions.ID)
    private Long id;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }
}

File: core-api/src/main/java/com/sequenceiq/cloudbreak/api/model/TemplateResponse.kt
package com.sequenceiq.cloudbreak.api.model

import com.sequenceiq.cloudbreak.doc.ModelDescriptions
import io.swagger.annotations.ApiModel
import io.swagger.annotations.ApiModelProperty

@ApiModel
class TemplateResponse : TemplateBase() {
    @ApiModelProperty(ModelDescriptions.ID)
    var id: Long? = null
}
